config.json

```diff
@@ -27,6 +27,14 @@
               }
             ]
           }
+        },
+        "minSearchLength": {
```

Comment:
Can we rename this to `minimalSearchTermLength`? If it is possible always be as descriptive as possible without removing parts that may be important to know. This only applies if it makes sense and shouldn't be done for naming that is already common sense.

That one is more of a gut feeling though, as "search" could be synonymous with "query" or "search term", so I am up for an argument, in case you want to leave it as it is.